File: src/main/Java/boj_step15/boj4134.java
import java.io.*;
import java.math.BigInteger;

public class boj4134 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        while (n-- > 0) {
            BigInteger x = new BigInteger(reader.readLine());
            if (x.isProbablePrime(10)) writer.append(x + "\n");
            else writer.append(x.nextProbablePrime() + "\n");
        }
        writer.close();
    }
}

File: src/main/boj_step15/boj4134.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine().toInt()
    repeat(n) {
        val x = r.readLine().toBigInteger()
        if(x.isProbablePrime(10)) w.write(x.toString())
        else w.write(x.nextProbablePrime().toString())
        w.newLine()
    }
    w.close()
}
